dialects/debug/debug.cpp

```diff
@@ -0,0 +1,38 @@
+#include "dialects/debug/debug.h"
+
+#include <thorin/config.h>
+#include <thorin/pass/pass.h>
+
+#include "thorin/dialects.h"
+
+#include "dialects/debug/passes/remove_perm.h"
+
+using namespace thorin;
+
+/// heart of the dialect
+/// registers passes in the different optimization phases
+/// as well as normalizers for the axioms
+extern "C" THORIN_EXPORT thorin::DialectInfo thorin_get_dialect_info() {
+    return {"debug",
+            [](thorin::PipelineBuilder& builder) {
+                builder.extend_codegen_prep_phase([](PassMan& man) { man.add<thorin::debug::DebugRemovePerm>(); });
+            },
+            nullptr, [](Normalizers& normalizers) { debug::register_normalizers(normalizers); }};
+}
+
+#define YELLOW "\033[0;33m"
+#define BLANK  "\033[0m"
```

Comment:
I believe this doesn't work on Win, and I think we have some preprocessing macro for this at some other place, but since I don't use Win I don't care so much XD
So, unless you want to look into it, I'd suggest to simply keep it like this.